src/index.js

```diff
@@ -42,4 +45,17 @@ const param = (currentList, currentIndex, li) => {
     }
```

Comment:
- [OPTIONAL ] The approach you used to store data in local storage is a valid and common one. However, another professional logic to store data in local storage is to use `JSON.stringify()` and `JSON.parse()` methods to convert the data into a string before storing it and then parse it back into an object when retrieving it from local storage. :wink:  This can be another good way of doing that :smile: 
- 
```
// Store data in local storage
const setLocalStorage = (list) => {
  localStorage.setItem('toDoList', JSON.stringify(list));
};

// Retrieve data from local storage
const readLocalStorage = () => {
  const list = JSON.parse(localStorage.getItem('toDoList'));
  return list || [];
};
```

Reply:
Thank you so much for recommending another perspective. I will refactor my code in the upcoming week

Reply:
Nice @UbdaNam  you did a great work :clap: Keep it up